diff --git a/.spacemacs b/.spacemacs
index b1b6f0d..509ecab 100644
--- a/.spacemacs
+++ b/.spacemacs
@@ -60,8 +60,7 @@ before layers configuration."
    ;; List of themes, the first of the list is loaded when spacemacs starts.
    ;; Press <SPC> T n to cycle to the next theme in the list (works great
    ;; with 2 themes variants, one dark and one light)
-   dotspacemacs-themes '(solarized-light
-                         solarized-dark
+   dotspacemacs-themes '(
                          leuven
                          monokai
                          zenburn)
diff --git a/fix-esup.el b/fix-esup.el
index 33fbefc..5e721cc 100644
--- a/fix-esup.el
+++ b/fix-esup.el
@@ -1 +1,10 @@
 ;;
+
+(let ((base "~/.emacs.d/elpa/"))
+  (add-to-list 'load-path base)
+  (dolist (f (directory-files base))
+    (let ((name (concat base "/" f)))
+      (when (and (file-directory-p name)
+                 (not (equal f ".."))
+                 (not (equal f ".")))
+        (add-to-list 'load-path name)))))
\ No newline at end of file
diff --git a/lisp/eab-dired.el b/lisp/eab-dired.el
index d62dfb7..7c12a29 100644
--- a/lisp/eab-dired.el
+++ b/lisp/eab-dired.el
@@ -8,23 +8,23 @@
 ;; Status: not intended to be distributed yet
 
 
-(eab/patch-this-code
- 'dired-async-create-files
- "(require 'cl-lib) (require 'dired-aux) (require 'dired-x)"
- (let ((print-quoted 't))
-   (prin1-to-string
-    `(progn
-       (require 'cl-lib)
-       (require 'dired-aux)
-       (require 'dired-x)
-       (add-to-list
-	'load-path
-	,(file-name-directory
-	  (buffer-file-name
-	   (car (find-function-noselect 'docker-tramp-add-method)))))
-       (require 'tramp)
-       (add-to-list 'tramp-methods ',eab/sussh)
-       (require 'docker-tramp)))) 't)
+;; (eab/patch-this-code
+;;  'dired-async-create-files
+;;  "(require 'cl-lib) (require 'dired-aux) (require 'dired-x)"
+;;  (let ((print-quoted 't))
+;;    (prin1-to-string
+;;     `(progn
+;;        (require 'cl-lib)
+;;        (require 'dired-aux)
+;;        (require 'dired-x)
+;;        (add-to-list
+;; 	'load-path
+;; 	,(file-name-directory
+;; 	  (buffer-file-name
+;; 	   (car (find-function-noselect 'docker-tramp-add-method)))))
+;;        (require 'tramp)
+;;        (add-to-list 'tramp-methods ',eab/sussh)
+;;        (require 'docker-tramp)))) 't)
 
 (dired-async-mode)
 (dired-async--modeline-mode 0)
diff --git a/lisp/eab-ido.el b/lisp/eab-ido.el
index fd4ea46..529a1e9 100644
--- a/lisp/eab-ido.el
+++ b/lisp/eab-ido.el
@@ -57,19 +57,19 @@
     (eab/cxb-1)
     (setq wg-dissociate-buffer-on-kill-buffer tmp)))
 
-;; Disable. Check speed
-(eab/patch-this-code
- 'projectile-project-buffer-p
- "(file-truename default-directory)" "(expand-file-name default-directory)")
-
+;; ;; Disable. Check speed
 ;; (eab/patch-this-code
-;;  'projectile-file-truename
-;;  "(file-truename file-name)" "(expand-file-name file-name)")
+;;  'projectile-project-buffer-p
+;;  "(file-truename default-directory)" "(expand-file-name default-directory)")
+
+;; ;; (eab/patch-this-code
+;; ;;  'projectile-file-truename
+;; ;;  "(file-truename file-name)" "(expand-file-name file-name)")
 
-;; Disable. Check speed
-(eab/patch-this-code
- 'projectile-project-root
- "(file-truename dir)" "(expand-file-name dir)")
+;; ;; Disable. Check speed
+;; (eab/patch-this-code
+;;  'projectile-project-root
+;;  "(file-truename dir)" "(expand-file-name dir)")
 
 ;; TODO добавить в список буферов "projectile-switch-project",
 ;; можно также использовать projectile-discover-projects-in-directory
diff --git a/lisp/eab-smex.el b/lisp/eab-smex.el
index 31a1a9e..b6b2da9 100644
--- a/lisp/eab-smex.el
+++ b/lisp/eab-smex.el
@@ -40,7 +40,7 @@
   (smex-update-and-rerun))
 
 ;; DONE убрать грубое внедрение в код smex.el
-(eab/patch-this-code 'smex-completing-read "(car choices)" "")
+;; (eab/patch-this-code 'smex-completing-read "(car choices)" "")
 
 (defun eab/smex-extended ()
   "Command for calling `helm-minibuffer-history' from `smex'"
